requirements.txt

```diff
@@ -1,6 +1,5 @@
 conan>=1.40.1, <2.00.0
 pandas
-parse>=1.18.0
```

Comment:
I can confirm that `parse` is no longer needed as a required package.  However, can you please split out this commit into a separate commit?  Right now this change is not related to the rest of the commit.